workflows/ai_surgical_video/python/ai_surgical_video.py

```diff
@@ -74,6 +75,27 @@ def compute(self, op_input, op_output, context):
         op_output.emit(in_message, "out")
 
 
+class DecimationOp(Operator):
```

Comment:
Maybe call it something else than DecimationOp (makes me think of 3D decimation)?

Reply:
Changed it to `FrameSamplerOp`, is it better?

Reply:
Perfect. Can you fix the linting and we merge